code/__DEFINES/traits/declarations.dm

```diff
@@ -555,6 +555,8 @@ Remember to update _globalvars/traits.dm if you're adding/removing/renaming trai
 
 /// This mob is phased out of reality from magic, either a jaunt or rod form
 #define TRAIT_MAGICALLY_PHASED "magically_phased"
+/// flips the direction of client movement
+#define TRAIT_INVERTED_MOVEMENT "inverted_movement"
```

Comment:
Funnily enough you can do this without a trait (IE, by using an element). 

You can hook `COMSIG_MOB_CLIENT_PRE_MOVE` and do `move_args[MOVE_ARG_DIRECTION] = REVERSE_DIR(move_args[MOVE_ARG_DIRECTION])`. We use this for the confusion status effect. 

You don't have to do that in this pr, but just a neat idea to reduce the surface area of this.

Reply:
Done.